<launch>

<node name="pressure_altimeter" pkg="pressure_altimeter"
    type="pressure_altimeter" output="screen">
    <!-- name of the frame in the header of the output -->
    <param name="world_frame_id" type="string" value="/world"/>
    <remap from="~pressure" to="/imu_3dm_gx4/pressure"/>
</node>	
  	
 <node name="attitude_eskf" pkg="kr_attitude_eskf" type="kr_attitude_eskf"
  output="screen">
  <param name="enable_magnetometer" type="bool" value="true"/>
  <param name="calibrate_magnetometer" type="string" value="none"/>
  <param name="broadcast_frame" type="bool" value="false"/>
  <param name="publish_pose" type="bool" value="false"/>
  <remap from="~imu" to="/imu_3dm_gx4/imu"/>
  <remap from="~magnetic_field" to="/imu_3dm_gx4/magnetic_field"/>
  <!-- Load filter parameters from galt_setup -->
  <!-- <rosparam file="$(find galt_setup)/config/params_3dm_gx4_25_post.yaml"/> -->
</node> 	

 
  <node name="gps_odom" pkg="gps_odom" type="gps_odom" output="screen">
    <remap from="imu" to="/attitude_eskf/filtered_imu"/>
    <remap from="pressure_height" to="/pressure_altimeter/height"/>
    <remap from="fix" to="/djiros/gps"/>
    <remap from="fix_velocity" to="/djiros/velo"/>

    <param name="world_frame_id" type="string" value="/world"/>
    <param name="body_frame_id" type="string" value="/body"/>
  </node>
</launch>
